apps/web/src/pages/index.tsx

```diff
@@ -1,23 +1,73 @@
 import { Layout } from "../helper/layout";
 import type { ReactElement } from "react";
+import Image from "next/image";
+import Link from "next/link";
 
 export default function Landing() {
   return (
-    <>
+    <div className="flex flex-col gap-9 h-[calc(100vh-6rem)] justify-between content-start">
       <div>
-        <h1 className="text-9xl font-sora">
+        <h1 className="text-[10rem] font-sora -tracking-[11.75px] break-normal">
           <span>SK</span>
-          <span className="text-techdev-gradient-radial">TechDev</span>
+          <wbr />
+          <span className="text-techdev-gradient-radial font-bold">
+            Tech
+            <wbr />
+            Dev
+          </span>
         </h1>
```

Comment:
This has too much line height. You can see this the clearest on mobile.
To save you some time, it's 128px.